assets/main.js

```diff
@@ -63,8 +69,15 @@ async function submitWord() {
   if (wordUser != wordDay) {
     checkContainLetters(lettersNow, wordDay);
     checkLettersPosition(lettersNow, wordDay);
+
+    if (attemptsCounter === ATTEMPTS_LIMIT) {
```

Comment:
Change to style guide project

```suggestion
    if (attemptsCounter === attemptsLimit) {
```